modelz/console.py

```diff
@@ -30,3 +32,13 @@ def print(self, msg: str):
 
 
 console = ModelzConsole() if config.disable_rich else Console()
+
+
+def jsonFormattedPrint(resp: Response):
+    if hasattr(resp.parsed, "to_dict") and callable(resp.parsed.to_dict):
+        formatted = json.dumps(resp.parsed.to_dict(), indent=2)
+    elif resp.parsed is None:
+        formatted = "OK"
+    else:
+        formatted = resp.content.decode(errors="ignore")
+    console.print(formatted)
```

Comment:
If the purpose is to pretty print the JOSN body, this is a bit hack since the interface of `Response` is ambiguous. Maybe we just need to print the `resp.parsed or resp.content`. I'm not sure if we can use "OK" when `reps.parsed is None`.